js/components/Marathon.js

```diff
@@ -246,11 +246,13 @@ var Marathon = React.createClass({
         wait: true
       });
     }
+    /*eslint-enable no-alert */
   },
 
   restartApp: function () {
     var app = this.state.activeApp;
 
+    /*eslint-disable no-alert, no-console */
     if (confirm("Restart app '" + app.id + "'?")) {
```

Comment:
It would be good to create a utility that handles confirms.
So we could do `User.confirm` or `Modal.confirm` or if we can't think of a good name, just put it in `Util.confirm` :p
That way we only have to disable `eslint` in one place rather than multiple places.